src/helpers/BlizzardApi.js

```diff
@@ -46,6 +46,14 @@ class BlizzardApi { // TODO: extends ExternalApi that provides a generic _fetch
     });
   }
 
+  async fetchClassicCharacter(regionCode, realm, name) {
```

Comment:
quick explanatory comment on why the forum is used.

```suggestion
  /**
   * Blizzard does not expose an Armory for Classic characters, but they expose structured data from the forum about the character, including a thumbnail (which is what we mostly care about).
   *
   * This approach is taken from the method that WCL uses for claiming characters and collecting thumbnails for Classic.
   */
  async fetchClassicCharacter(regionCode, realm, name) {
```

Reply:
lgtm 👍